test/ps_inverse.jl

```diff
@@ -5,7 +5,10 @@
     for d in 1:5
         S = Nemo.MatrixSpace(T, d, d)
         for case in 1:20
-            M = 1+S([random_ps(T) for i in 1:d, j in 1:d])
+            M = S([random_ps(T) for i in 1:d, j in 1:d])
+            while isequal(ps_matrix_const_term(M), 0)
```

Comment:
I do not think this will solve the problem completely: you may have a nonzero but singular matrix as a constant term

Reply:
Added `det` check instead